courses/tests.py

```diff
@@ -0,0 +1,22 @@
+from django.test import TestCase
```

Comment:
Can you create a directory for tests and rename this something like `test_json_list_field.py` or `json_list_field_tests.py`?